assets/results.js

```diff
@@ -0,0 +1,84 @@
+
+// Checks document location for values passed from homepage.js
+// Runs function for either name, ingredient, or random
+function SliderCheck() {
```

Comment:
can we rename this to something like searchDrink? It's a little misleading since it's not just checking the slider.